src/core/public/chrome/chrome_service.tsx

```diff
@@ -220,7 +220,7 @@ export class ChromeService {
                 >
                   <FormattedMessage
                     id="core.chrome.browserDeprecationLink"
-                    defaultMessage="the support matrix on our website"
+                    defaultMessage="our supported browsers list on our website"
```

Comment:
```suggestion
                    defaultMessage="the supported browsers list on our website"
```

nit to avoid repeating "our".

Reply:
The grammar will still be off. Lemme try this.